cmd/bucket-handlers.go

```diff
@@ -484,7 +499,12 @@ func (api objectAPIHandlers) DeleteMultipleObjectsHandler(w http.ResponseWriter,
 
 	// Write success response.
 	writeSuccessResponseXML(w, encodedSuccessResponse)
-
+	for _, o := range deletedObjects {
```

Comment:
I don't entirely understand the logic for populating the deletedObjects array, it currently gets populated for `""` `"NoSuchKey"` and `"InvalidArgument"`

Reply:
DeleteObjects API could receive objects which no longer exist/invalid version - in such cases, error is just ignored.See [S3 Spec](https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteObjects.html)